javascript/javascript2/week1/main.js

```diff
@@ -0,0 +1,26 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+function renderProducts(products) {
+    let listUl = document.createElement('ul');
+    let main = document.querySelector('main');
+    let listLi;
+    main.appendChild(listUl);
+    for (let i = 0; i < products.length; i++) {
+        listLi = document.createElement('li');
+        listLi.innerHTML = `<b>${products[i].name}</b> <br>  
+            Price: ${products[i].price} <br>
+            Rating: ${products[i].rating}`;
+        listUl.appendChild(listLi);
+    }
+}
+
+renderProducts(products); 
+
+
+
+
+
+
```

Comment:
Clean up these extra lines. Clean code is easier to read.

Reply:
ok I will